androidjava/developer-guide/faqs/_index.md

```diff
@@ -33,12 +47,16 @@ In general, Aspose.Slides for Android via Java can easily handle presentation fi
 
 **A**: You can render the presentation slides to images and can use these images for previewing the slides.
 
-## **Working with Presentations**
+## **Working with Text**
 
 **Q: Is it possible to retrieve all the text from a presentation?**
 
 **A**: Aspose.Slides for Android via Java provides the [SlideUtil](https://reference.aspose.com/slides/androidjava/com.aspose.slides/slideutil/) class that provides various methods for retrieving whole text from the presentations.
 
+**Q: Why are paragraph sizes different on Windows and Linux operating systems?**
```

Comment:
@andreypotapovaspose 

I think it's worth changing the name to reflect the specifics of the product, for example to something like this:
Why are paragraph sizes different on PC and Android?

Reply:
@petrtetenko,
Fixed.